scene/gui/code_edit.cpp

```diff
@@ -1886,6 +1882,55 @@ void CodeEdit::request_code_completion(bool p_force) {
 		emit_signal(SNAME("code_completion_requested"));
 	} else if (ofs > 1 && line[ofs - 1] == ' ' && code_completion_prefixes.has(line[ofs - 2])) {
 		emit_signal(SNAME("code_completion_requested"));
+	} else {
+		// try to find an opening bracket indicating we are between two brackets.
+		int stack = 1;
+		bool found_match = false;
+		char32_t closec = '(';
+		char32_t c = ')';
+		for (int i = get_caret_line(); i >= 0 && !found_match; i--) {
```

Comment:
Should use `CodeEdit` built in string and comment management here.

Wondering if we should do this for all `auto_brace_completion_pairs` rather then hard coding it to `( )`.